lib/resources/datav2/websockets_v2.js

```diff
@@ -195,39 +207,48 @@ class AlpacaStreamV2Client extends events.EventEmitter {
       this.session.subscriptions.trades.filter(
         (trade) => !trades.includes(trade)
       );
-    this.unsubscribe(trades, [], [], []);
+    this.unsubscribe(trades, [], [], [], []);
   }
 
   unsubscribeFromQuotes(quotes) {
     this.session.subscriptions.quotes =
       this.session.subscriptions.quotes.filter(
         (quote) => !quotes.includes(quote)
       );
-    this.unsubscribe([], quotes, [], []);
+    this.unsubscribe([], quotes, [], [], []);
   }
 
   unsubscribeFromBars(bars) {
     this.session.subscriptions.bars = this.session.subscriptions.bars.filter(
       (bar) => !bars.includes(bar)
     );
-    this.unsubscribe([], [], bars, []);
+    this.unsubscribe([], [], bars, [], []);
   }
 
   unsubscriceFromDailyBars(dailyBars) {
     this.session.subscriptions.dailyBars =
       this.session.subscriptions.dailyBars.filter(
         (dailyBar) => !dailyBars.includes(dailyBar)
       );
-    this.unsubscribe([], [], [], dailyBars);
+    this.unsubscribe([], [], [], dailyBars, []);
+  }
+
+  unsubscribeFromStatuses(statuses) {
+    this.session.subscriptions.statuses =
+      this.session.subscriptions.statuses.filter((status) => {
+        !statuses.includes(status);
+      });
```

Comment:
I am not sure this works, the inner fucntion has no return value. I think replace curly braces with parenthesis

Reply:
yep this would return `null` which gets evaluated as `false` and would therefore filter nothing
```suggestion
      this.session.subscriptions.statuses.filter((status) => !statuses.includes(status));
```